exercises/perfect-numbers/generator/canonical-data.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
This information should either be downloaded from GitHub or loaded from a local copy of the problem-specs repo. Generators exist to make it easier to update test suites when there are changes to the shared test data, inserting a fixed version here prevents that.